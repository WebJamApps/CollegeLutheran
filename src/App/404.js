
import React from 'react';

const FourOhFour = () => (
  <div className="page-content">
    <div className="fof">
          404 - Page not available
    </div>
  </div>
);
export default FourOhFour;
