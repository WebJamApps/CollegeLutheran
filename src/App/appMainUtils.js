import React from 'react';

const activeUsers = (heartBeat, userCount) => (
  <div>
    <button
      type="button"
      style={{
        cursor: 'none', marginLeft: '10px', height: '18px', width: '18px', backgroundColor: heartBeat,
      }}
    >
      {' '}
    </button>
    <span style={{ paddingLeft: '10px', marginTop: '10px' }}>
      <i>Active Users</i>
:
      {' '}
      {userCount}
    </span>
  </div>
);

export default { activeUsers };
