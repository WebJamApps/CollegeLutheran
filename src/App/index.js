import PropTypes from 'prop-types';
import 'core-js/stable';
import 'regenerator-runtime/runtime';
import superagent from 'superagent';
import React, { Component } from 'react';
import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
import { connect } from 'react-redux';
import DefaultMusic from '../containers/Music';
import Beliefs from '../containers/Beliefs';
import DefaultFamily from '../containers/Family';
import Giving from '../containers/Giving';
import Staff from '../containers/Staff';
import DefaultYouth from '../containers/Youth';
import AdminDashboardDefault from '../containers/AdminDashboard';
import DefaultNews from '../containers/News';
import Calendar from '../containers/Calendar';
import AppFourOhFour from './404';
import AppTemplateDefault from './AppTemplate';
import DefaultHome from '../containers/Homepage';
import mapStoreToProps from '../redux/mapStoreToProps';
import fetch from './fetch';

export class App extends Component {
  constructor(props) {
    super(props);
    this.fetch = fetch;
    this.state = {};
    this.superagent = superagent;
  }

  componentDidMount() { // fetch the books to populate homepage content, youth pics, and children pics
    this.fetch(this, 'book/one?type=homePageContent', 'GOT_HOMEPAGE');
    this.fetch(this, 'book?type=familyPics', 'GOT_FAMILYPICS');
    this.fetch(this, 'book?type=Forum', 'GOT_BOOKS');
    this.fetch(this, 'book?type=youthPics', 'GOT_YOUTHPICS');
  }

  render() {
    const { auth } = this.props;
    const userRoles = JSON.parse(process.env.userRoles).roles;
    return (
      <div id="App" className="App">
        <Router>
          <AppTemplateDefault id="homepage">
            <Switch>
              <Route exact path="/" component={DefaultHome} />
              <Route path="/music" component={DefaultMusic} />
              <Route path="/belief" component={Beliefs} />
              <Route path="/family" component={DefaultFamily} />
              <Route path="/giving" component={Giving} />
              <Route exact path="/staff" component={Staff} />
              {auth.isAuthenticated && userRoles.indexOf(auth.user.userType) !== -1
                ? <Route path="/admin" component={AdminDashboardDefault} /> : null}
              <Route path="/youth" component={DefaultYouth} />
              <Route path="/news" component={DefaultNews} />
              <Route path="/calendar" component={Calendar} />
              <Route component={AppFourOhFour} />
            </Switch>
          </AppTemplateDefault>
        </Router>
      </div>

    );
  }
}
App.propTypes = {
  auth: PropTypes.shape({ user: PropTypes.shape({ userType: PropTypes.string }), isAuthenticated: PropTypes.bool }),
};
App.defaultProps = { auth: { isAuthenticated: false, user: { userType: '' } } };

export default connect(mapStoreToProps, null)(App);
