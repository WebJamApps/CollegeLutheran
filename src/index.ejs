<!DOCTYPE html>
<html>
  <head prefix="og: http://ogp.me/ns# fb: http://ogp.me/ns/fb#">
    <meta property="og:title" content="College Lutheran Church" />
    <meta property="og:description" content="We celebrate God's grace and share His love in Christ!">
    <meta property="og:image" content="https://upload.wikimedia.org/wikipedia/commons/thumb/a/ab/Lutherrose.svg/800px-Lutherrose.svg.png">
    <meta property="og:url" content="https://www.collegelutheran.org">
    <title>
      <%- htmlWebpackPlugin.options.metadata.title %>
    </title>
    <base href="<%- htmlWebpackPlugin.options.metadata.baseUrl %>" />
    <link
      href="https://fonts.googleapis.com/css?family=PT Sans Caption"
      rel="stylesheet"
    />
    <link href="https://fonts.googleapis.com/css?family=Habibi" rel="stylesheet" />
    <link
      href="https://fonts.googleapis.com/icon?family=Material+Icons"
      rel="stylesheet"
    />
    <link
      rel="stylesheet"
      href="https://use.fontawesome.com/releases/v5.8.1/css/all.css"
      integrity="sha384-50oBUHEmvpQ+1lW4y57PTFmhCaXp0ML5d60M1M7uH2+nqUivzIebhndOJK28anvf"
      crossorigin="anonymous"
    />

    <meta charset="utf-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1" />
  </head>
  <body>
    <div>
      <div id="root">
        <div class="splash">
          <div class="message">
            <%- htmlWebpackPlugin.options.metadata.title %>
          </div>
          <i class="fa fa-spinner fa-spin"></i>
        </div>
      </div>
    </div>
  </body>
</html>
