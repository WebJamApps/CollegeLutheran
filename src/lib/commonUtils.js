const setTitleAndScroll = (pageTitle) => {
  if (pageTitle !== '') pageTitle += ' | ';// eslint-disable-line no-param-reassign
  document.title = `${pageTitle}College Lutheran Church`;
  const top = document.getElementsByClassName('page-content')[0];
  if (top !== undefined && typeof top.scrollIntoView === 'function') {
    top.scrollIntoView();
  }
};

export default { setTitleAndScroll };
