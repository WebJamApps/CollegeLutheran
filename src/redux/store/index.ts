import { createStore, applyMiddleware } from 'redux';
import { persistStore, persistReducer } from 'redux-persist';
import thunk from 'redux-thunk';
import storageSession from 'redux-persist/lib/storage/session';
import allReducers from '../allReducers';

const persistConfig = {
  key: 'root',
  storage: storageSession,
  blacklist: ['sc', 'familyPics', 'youthPics', 'musicPics', 'habitatPics'],
};
const mWares = applyMiddleware(thunk);
const persistedReducer = persistReducer(persistConfig, allReducers);
const store = createStore(persistedReducer, mWares);
const persistor = persistStore(store);
export default { store, persistor };
