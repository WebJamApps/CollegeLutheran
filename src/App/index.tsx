import 'core-js/stable';
import 'regenerator-runtime/runtime';
import superagent from 'superagent';
import React, { Component } from 'react';
import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
import { connect } from 'react-redux';
import commonUtils from '../lib/commonUtils';
import DefaultMusic from '../containers/Music';
import Beliefs from '../containers/Beliefs';
import DefaultFamily from '../containers/Family';
import Giving from '../containers/Giving';
import Staff from '../containers/Staff';
import DefaultYouth from '../containers/Youth';
import AdminDashboardDefault from '../containers/AdminDashboard';
import DefaultNews from '../containers/News';
import Calendar from '../containers/Calendar';
import AppFourOhFour from './404';
import AppTemplateDefault from './AppTemplate';
import DefaultHome from '../containers/Homepage';
import mapStoreToProps from '../redux/mapStoreToProps';
import fetch from '../lib/fetch';
import { AppProps } from './AppTypes';

export class App extends Component<AppProps> {
  fetch: typeof fetch;

  superagent: superagent.SuperAgentStatic;

  static defaultProps = {
    dispatch: /* istanbul ignore next */(): void => { },
    auth: {
      isAuthenticated: false,
      user: { userType: '' },
      error: '',
      email: '',
      token: '',
    },
  };

  constructor(props: AppProps) {
    super(props);
    this.fetch = fetch;
    this.state = {};
    this.superagent = superagent;
  }

  componentDidMount(): void { // fetch the books to populate homepage content, youth pics, music pics and children pics
    this.fetch.fetchGet(this, 'book/one?type=homePageContent', 'GOT_HOMEPAGE');
    this.fetch.fetchGet(this, 'book?type=familyPics', 'GOT_FAMILYPICS');
    this.fetch.fetchGet(this, 'book?type=Forum', 'GOT_BOOKS');
    this.fetch.fetchGet(this, 'book?type=youthPics', 'GOT_YOUTHPICS');
    this.fetch.fetchGet(this, 'book?type=otherPics', 'GOT_OTHERPICS');
    this.fetch.fetchGet(this, 'book?type=musicPics', 'GOT_MUSICPICS');
  }

  render(): JSX.Element {
    const { auth } = this.props;
    const userRoles: string[] = commonUtils.getUserRoles();
    return (
      <div id="App" className="App">
        <Router>
          <AppTemplateDefault>
            <Switch>
              <Route exact path="/" component={DefaultHome} />
              <Route path="/music" component={DefaultMusic} />
              <Route path="/belief" component={Beliefs} />
              <Route path="/family" component={DefaultFamily} />
              <Route path="/giving" component={Giving} />
              <Route exact path="/staff" component={Staff} />
              {auth.isAuthenticated && auth.user.userType && userRoles.indexOf(auth.user.userType) !== -1
                ? <Route path="/admin" component={AdminDashboardDefault} /> : null}
              <Route path="/youth" component={DefaultYouth} />
              <Route path="/news" component={DefaultNews} />
              <Route path="/calendar" component={Calendar} />
              <Route component={AppFourOhFour} />
            </Switch>
          </AppTemplateDefault>
        </Router>
      </div>

    );
  }
}

export default connect(mapStoreToProps, null)(App);
