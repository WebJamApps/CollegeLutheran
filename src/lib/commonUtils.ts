import { Ibook } from '../redux/mapStoreToProps';
import type { Homepage } from '../containers/Homepage/index';

const setTitleAndScroll = (pageTitle: string, width?: number): void => {
  if (pageTitle !== '') pageTitle += ' | ';// eslint-disable-line no-param-reassign
  document.title = `${pageTitle}College Lutheran Church`;
  let getClass = 'page-content';
  if (width !== undefined && width < 1004) getClass = 'material-header';
  const top = document.getElementsByClassName(getClass)[0];
  if (top !== undefined && typeof top.scrollIntoView === 'function') top.scrollIntoView();
};

async function randomizePics(view: Homepage, delay: () => void): Promise<void> {
  await delay();
  const {
    familyPics, youthPics, otherPics, musicPics,
  } = view.props;
  let arr: Ibook[] | never[] = familyPics || [];
  const arr2: Ibook[] = youthPics || [];
  const arr3: Ibook[] = musicPics || [];
  arr = arr.concat(otherPics || []).concat(arr2).concat(arr3);

  for (let i = arr.length - 1; i > 0; i -= 1) {
    const j = Math.floor(Math.random() * (i + 1));
    [arr[i], arr[j]] = [arr[j], arr[i]];// eslint-disable-line security/detect-object-injection
  }
  view.setState({ picsState: arr });
}

function getUserRoles(): string[] {
  let userRoles: string[] = [];
  try {
    userRoles = JSON.parse(process.env.userRoles || /* istanbul ignore next */'').roles;
  } catch (e) { /* istanbul ignore next */userRoles = []; }
  return userRoles;
}

export default {
  getUserRoles, setTitleAndScroll, randomizePics,
};
