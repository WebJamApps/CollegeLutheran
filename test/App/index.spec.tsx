import renderer from 'react-test-renderer';
import store from 'src/redux/store';
import { App, showAdminDashboard, checkIsAdmin } from 'src/App';
import { Provider } from 'react-redux';
import commonUtils from 'src/lib/commonUtils';

describe('App component', () => {
  it('is defined', () => {
    expect(App).toBeDefined();
  });
  it('renders the component', () => {
    commonUtils.setTitleAndScroll = jest.fn();
    const href = 'http://localhost:7777';
    const reload = jest.fn();
    Object.defineProperty(window, 'location', {
      value: {
        href, assign: () => { }, reload, origin: href,
      },
      writable: true,
    });
    const app = renderer.create(<Provider store={store.store}><App /></Provider>).toJSON();
    expect(app).toMatchSnapshot();
  });
  it('showAdminDashboard when isAdmin', () => {
    const result = showAdminDashboard(true);
    expect(result).not.toBe(null);
  });
  it('checkIsAdmin', () => {
    const setIsAdmin = jest.fn();
    process.env.userRoles = JSON.stringify({ roles: ['tester'] });
    checkIsAdmin({
      error: '', token: 'token', isAuthenticated: true, user: { userType: 'tester', email: 'tester@tesing.com' },
    }, setIsAdmin);
    expect(setIsAdmin).toHaveBeenCalledWith(true);
  });
});
