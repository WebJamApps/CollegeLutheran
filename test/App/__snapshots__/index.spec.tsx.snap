// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component renders the component 1`] = `
<div
  className="App"
  id="App"
>
  <div
    className="page-host"
  >
    <div
      className="home-sidebar close drawer-container"
      id="sidebar"
      onClick={[Function]}
      onKeyPress={[Function]}
      role="button"
      tabIndex={0}
    >
      <div
        className="drawer"
        style={
          {
            "backgroundColor": "#c0c0c0",
            "position": "relative",
            "zIndex": -1,
          }
        }
      >
        <div
          className="navImage"
        >
          <img
            alt="Luther Rose"
            id="webjamwidelogo"
            src="https://upload.wikimedia.org/wikipedia/commons/thumb/a/ab/Lutherrose.svg/800px-Lutherrose.svg.png"
            style={
              {
                "marginLeft": 0,
                "marginRight": 0,
                "width": "86px",
              }
            }
          />
        </div>
        <div
          className="nav-list"
          style={
            {
              "width": "220px",
            }
          }
        >
          <p
            style={
              {
                "fontSize": "1px",
                "marginBottom": "2px",
              }
            }
          />
          <div
            className="menu-item"
            style={
              {
                "backgroundColor": "#244a8bff",
              }
            }
          >
            <p
              style={
                {
                  "color": "#fff",
                  "fontSize": "11pt",
                  "marginBottom": "2px",
                }
              }
            >
              <a
                className="menu-hover"
                href="http://bit.ly/CollegeLutheranDirections"
                style={
                  {
                    "color": "#88c1ff",
                  }
                }
              >
                <span>
                  210 S. College Ave
                </span>
              </a>
              <br />
              Salem, VA 24153
            </p>
          </div>
          <div
            className="menu-item"
            style={
              {
                "backgroundColor": "#244a8bff",
                "marginLeft": "0px",
              }
            }
          >
            <p
              style={
                {
                  "color": "#fff",
                  "fontSize": "11pt",
                  "marginBottom": "2px",
                }
              }
            >
              <span>
                ph: 
              </span>
              <a
                className="menu-hover"
                href="tel:5403894963"
                style={
                  {
                    "color": "#88c1ff",
                  }
                }
              >
                (540) 389-4963
              </a>
              <br />
              <span>
                fax: 
              </span>
              (540) 389-4980
              <br />
              <a
                href="mailto:office1@collegelutheran.org"
                style={
                  {
                    "color": "#88c1ff",
                    "wordWrap": "break-word",
                  }
                }
              >
                <span
                  className="menu-hover"
                >
                  office1@collegelutheran.org
                </span>
              </a>
            </p>
          </div>
           
          <div
            className="menu-item"
          >
            <a
              className="nav-link"
              href="/belief"
              onClick={[Function]}
            >
              <div
                style={
                  {
                    "display": "inline",
                  }
                }
              >
                <i
                  className="fas fa-arrows-alt"
                />
                 
                <span
                  className="nav-item"
                >
                  Our Lutheran Beliefs
                </span>
              </div>
            </a>
          </div>
          <div
            className="menu-item"
          >
            <a
              className="nav-link"
              href="/staff"
              onClick={[Function]}
            >
              <div
                style={
                  {
                    "display": "inline",
                  }
                }
              >
                <i
                  className="fas fa-users"
                />
                 
                <span
                  className="nav-item"
                >
                  Church Staff
                </span>
              </div>
            </a>
          </div>
          <div
            className="menu-item"
          >
            <a
              className="nav-link"
              href="/livestream"
              onClick={[Function]}
            >
              <div
                style={
                  {
                    "display": "inline",
                  }
                }
              >
                <i
                  className="fas fa-video"
                />
                 
                <span
                  className="nav-item"
                >
                  Livestream
                </span>
              </div>
            </a>
          </div>
          <div
            className="menu-item"
          >
            <a
              className="nav-link"
              href="/stewardship"
              onClick={[Function]}
            >
              <div
                style={
                  {
                    "display": "inline",
                  }
                }
              >
                <i
                  className="fas fa-step-forward"
                />
                 
                <span
                  className="nav-item"
                >
                  Stewardship
                </span>
              </div>
            </a>
          </div>
          <div
            className="menu-item"
          >
            <a
              className="nav-link"
              href="/news"
              onClick={[Function]}
            >
              <div
                style={
                  {
                    "display": "inline",
                  }
                }
              >
                <i
                  className="far fa-newspaper"
                />
                 
                <span
                  className="nav-item"
                >
                  News
                </span>
              </div>
            </a>
          </div>
          <div
            className="menu-item"
          >
            <a
              className="nav-link"
              href="/giving"
              onClick={[Function]}
            >
              <div
                style={
                  {
                    "display": "inline",
                  }
                }
              >
                <i
                  className="fas fa-handshake"
                />
                 
                <span
                  className="nav-item"
                >
                  Giving
                </span>
              </div>
            </a>
          </div>
          <div
            className="menu-item"
          >
            <a
              className="nav-link"
              href="/habitatproject"
              onClick={[Function]}
            >
              <div
                style={
                  {
                    "display": "inline",
                  }
                }
              >
                <i
                  className="fas fa-hammer"
                />
                 
                <span
                  className="nav-item"
                >
                  Habitat Project
                </span>
              </div>
            </a>
          </div>
          <div
            className="menu-item"
          >
            <a
              className="nav-link"
              href="/music"
              onClick={[Function]}
            >
              <div
                style={
                  {
                    "display": "inline",
                  }
                }
              >
                <i
                  className="fas fa-music"
                />
                 
                <span
                  className="nav-item"
                >
                  Music
                </span>
              </div>
            </a>
          </div>
          <div
            className="menu-item"
          >
            <a
              className="nav-link"
              href="/youth"
              onClick={[Function]}
            >
              <div
                style={
                  {
                    "display": "inline",
                  }
                }
              >
                <i
                  className="fas fa-futbol"
                />
                 
                <span
                  className="nav-item"
                >
                  Youth Ministry
                </span>
              </div>
            </a>
          </div>
          <div
            className="menu-item"
          >
            <a
              className="nav-link"
              href="/family"
              onClick={[Function]}
            >
              <div
                style={
                  {
                    "display": "inline",
                  }
                }
              >
                <i
                  className="fas fa-child"
                />
                 
                <span
                  className="nav-item"
                >
                  Children & Families
                </span>
              </div>
            </a>
          </div>
          <div
            className="menu-item"
          >
            <a
              className="nav-link"
              href="/"
              onClick={[Function]}
            >
              <div
                style={
                  {
                    "display": "inline",
                  }
                }
              >
                <i
                  className="fas fa-heart"
                />
                 
                <span
                  className="nav-item"
                >
                  Home
                </span>
              </div>
            </a>
          </div>
           
          <p
            style={
              {
                "fontSize": "6pt",
                "margin": 0,
                "padding": 0,
              }
            }
          >
             
          </p>
        </div>
      </div>
    </div>
    <div
      className="main-panel"
    >
      <span
        id="mobilemenutoggle"
        onClick={[Function]}
        onKeyPress={[Function]}
        role="button"
        tabIndex={0}
      >
        <i
          className="fas fa-bars"
        />
      </span>
      <div
        className="mainPanel"
      >
        <div
          className="swipe-area"
        />
        <div
          className="material-header home-header"
          id="header"
          style={
            {
              "backgroundColor": "#244a8bff",
            }
          }
        >
          <div
            className="headercontent"
          />
          <div>
            <div
              className="flex-header"
              style={
                {
                  "marginLeft": "15px",
                  "marginTop": "-18px",
                  "paddingLeft": "10px",
                  "paddingRight": "10px",
                }
              }
            >
              <h2
                className="header-text"
                style={
                  {
                    "fontSize": "25px",
                    "marginBottom": "-1px",
                    "marginTop": "1px",
                  }
                }
              >
                <a
                  className="header-text"
                  href="/"
                  style={
                    {
                      "textAlign": "left",
                      "textDecoration": "none",
                    }
                  }
                >
                  College Lutheran Church
                </a>
              </h2>
              <p
                className="subTitle"
                style={
                  {
                    "color": "#f4c00eff",
                    "maxWidth": "100%",
                  }
                }
              >
                We celebrate God's grace and share His love in Christ!
              </p>
            </div>
          </div>
        </div>
        <div
          className="content-block"
          id="contentBlock"
          style={
            {
              "width": "auto",
            }
          }
        >
          <div>
            <div
              className="page-content"
            >
              <div
                className="aboutPage"
              >
                <div
                  className="container-fluid"
                  style={
                    {
                      "paddingRight": 0,
                    }
                  }
                >
                  <div
                    className="row"
                  >
                    <div
                      className="col"
                      style={
                        {
                          "maxWidth": "7.35in",
                          "paddingRight": "4px",
                          "top": "0",
                        }
                      }
                    >
                      <p
                        style={
                          {
                            "fontSize": "18px",
                            "marginBottom": "40px",
                            "marginTop": "40px",
                          }
                        }
                      >
                        College Lutheran Church is located in Southwest Virginia in the beautiful city of Salem, right next to Roanoke, VA. The church is situated on College Avenue, within easy walking distance of Roanoke College. College Lutheran Church is part of the Evangelical Lutheran Church in America (ELCA).
                      </p>
                      <h5
                        style={
                          {
                            "fontWeight": "bold",
                            "marginTop": "35px",
                          }
                        }
                      />
                      <section
                        style={
                          {
                            "marginBottom": "35px",
                            "marginTop": "20px",
                            "textAlign": "left",
                          }
                        }
                      />
                      <p
                        style={
                          {
                            "marginBottom": "16px",
                            "paddingBottom": 0,
                            "paddingRight": "15px",
                          }
                        }
                      >
                        <span
                          style={
                            {
                              "fontSize": "18px",
                            }
                          }
                        >
                           
                          <strong>
                            What Else is Happening at CLC?
                          </strong>
                        </span>
                      </p>
                      <p>
                        Click
                         
                        <a
                          href="/news"
                        >
                          News
                        </a>
                         
                        to view or subscribe to our emails. Scroll below to view our CLC Events Calendar, and follow us on social media.
                        <br />
                      </p>
                    </div>
                    <div
                      className="col"
                      style={
                        {
                          "padding": "1px",
                          "paddingRight": "0",
                        }
                      }
                    >
                      <div
                        id="slideshow1"
                        style={
                          {
                            "margin": "auto",
                            "marginTop": "40px",
                            "textAlign": "center",
                          }
                        }
                      >
                        <img
                          alt="churchBuilding"
                          src="https://dl.dropboxusercontent.com/s/8wcnwvc7s9iclj5/clcBuilding.png?dl=0"
                          style={
                            {
                              "borderRadius": "50%",
                              "width": "100%",
                            }
                          }
                        />
                      </div>
                    </div>
                  </div>
                </div>
              </div>
              <hr />
              <div
                className="row"
                style={
                  {
                    "marginLeft": "10px",
                  }
                }
              >
                <div
                  className="col"
                  style={
                    {
                      "margin": "auto",
                      "maxWidth": "600px",
                      "paddingRight": "10px",
                    }
                  }
                >
                  <p
                    style={
                      {
                        "fontSize": "10pt",
                        "marginTop": "0px",
                        "textAlign": "center",
                      }
                    }
                  >
                    <a
                      href="/calendar"
                      style={
                        {
                          "fontSize": "10pt",
                        }
                      }
                    >
                      <i>
                        View Full Calendar
                      </i>
                    </a>
                  </p>
                  <iframe
                    frameBorder="0"
                    height="500"
                    scrolling="no"
                    src="https://calendar.google.com/calendar/embed?title=College%20Lutheran%20Events%20Calendar&showNav=0&showDate=0&showPrint=0&showTabs=0&showCalendars=0&mode=AGENDA&height=500&wkst=1&bgcolor=%23FFFFFF&src=ra8um2u26p59a2t92lfooq4els%40group.calendar.google.com&color=%23060D5E&ctz=America%2FNew_York"
                    style={
                      {
                        "borderWidth": 0,
                      }
                    }
                    title="google-calendar"
                    width="550"
                  />
                </div>
                <div
                  className="col"
                  style={
                    {
                      "margin": "auto",
                      "paddingLeft": "10px",
                      "paddingRight": "10px",
                      "textAlign": "center",
                      "width": "600px",
                    }
                  }
                >
                  <p
                    id="wideFacebook"
                    style={
                      {
                        "fontSize": "10pt",
                        "marginBottom": "0px",
                        "marginTop": "32px",
                        "textAlign": "center",
                      }
                    }
                  >
                    <i>
                      Like Us On
                       
                      <a
                        href="https://www.facebook.com/CollegeLutheranChurch/"
                        style={
                          {
                            "fontSize": "10pt",
                          }
                        }
                      >
                        Facebook
                      </a>
                    </i>
                  </p>
                  <iframe
                    allow="encrypted-media"
                    className="widescreenHomepage"
                    height="485"
                    scrolling="none"
                    src="//www.facebook.com/plugins/likebox.php?href=https%3A%2F%2Fwww.facebook.com%2FCollegeLutheranChurch&width=500&height=485&colorscheme=light&show_faces=false&header=true&stream=true&show_border=false"
                    style={
                      {
                        "border": "none",
                        "marginLeft": "-14px",
                        "overflow": "hidden",
                      }
                    }
                    title="facebook ticker"
                    width="500"
                  />
                </div>
              </div>
              <p
                style={
                  {
                    "fontSize": "6pt",
                    "marginBottom": "0",
                  }
                }
              >
                 
              </p>
            </div>
            <div
              style={
                {
                  "marginLeft": "-9px",
                  "textAlign": "center",
                }
              }
            >
              <a
                href="http://www.elca.org/"
                rel="noopener noreferrer"
                target="_blank"
              >
                <img
                  alt="ELCA LOGO"
                  id="elcaLogo"
                  src="https://dl.dropboxusercontent.com/s/wkzubcmmm3pqst4/elca-logo.png?dl=0"
                  style={
                    {
                      "margin": "auto",
                      "marginLeft": "",
                      "paddingTop": "30px",
                      "width": "308px",
                    }
                  }
                />
              </a>
            </div>
          </div>
          <div
            className="footer"
            id="wjfooter"
            style={
              {
                "bottom": "0",
                "marginTop": "20px",
                "paddingBottom": "20px",
                "paddingTop": "20px",
              }
            }
          >
            <div
              className="footer-links"
              style={
                {
                  "padding": "6px",
                  "textAlign": "center",
                }
              }
            >
              <a
                href="href="https://twitter.com/CollegeLutheran""
                rel="noopener noreferrer"
                style={
                  {
                    "paddingRight": "5px",
                  }
                }
                target="_blank"
              >
                <span>
                  <i
                    className="fab fa-twitter"
                  />
                </span>
              </a>
              <a
                href="https://www.facebook.com/CollegeLutheranChurch/"
                rel="noopener noreferrer"
                style={
                  {
                    "paddingRight": "5px",
                  }
                }
                target="_blank"
              >
                <span>
                  <i
                    className="fab fa-facebook"
                  />
                </span>
              </a>
              <a
                href="https://www.instagram.com/collegelutheranchurch/"
                rel="noopener noreferrer"
                style={
                  {
                    "paddingRight": "5px",
                  }
                }
                target="_blank"
              >
                <span>
                  <i
                    className="fab fa-instagram"
                  />
                </span>
              </a>
              <span
                className="nohover"
              >
                Powered by  
              </span>
              <a
                className="wjllc"
                href="https://www.web-jam.com"
                rel="noopener noreferrer"
                target="_blank"
              >
                Web Jam LLC
              </a>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
